Rovers/Rovers.tex

```diff
@@ -0,0 +1,405 @@
+SPEX could even show off the rover at the FIRST robotics competition (especially if the rover competed in the University Robotics Competition or a similar competition). 
+RIT would love the promotional value an it would be a great recruitment and fundraising opportunity for RIT SPEX.
+
+\subsection{Challenge}
+\label{subsec:challenge}
+The project including just a few of the improvements would be very ambitious. 
+It would require at a minimum a handful (4-5) members each with different areas of expertise just for the MVP.
+The team would have to be even larger to accommodate some of the improvements as a larger team would be able to accomplish more. 
+Aside from technical skills, members would need to be knowledge in \LaTeX{}, documentation, notetaking, and good research practices. 
+Challenging projects allow for members to gain even more experience. 
+
+\subsection{Documentation}
+\label{subsec:docs}
+It is important that the team members document the sources they use to gather information. 
+There will be a Google Drive folder that will hold notes with links to any books, articles, media or their sources that are relevant to the rover project. 
+The software and hardware will be tracked on GitHub (including the technical report).
+One of the areas SPEX is currently struggling with is knowledge transfer.
+This project should serve as an example to other SPEX projects on how information should be passed on from year to year.
+Being able to successfully transfer knowledge from year to year will greatly advance the ability for RIT SPEX to complete more ambitious projects.
+
+\subsection{Project Management Skills}
+\label{subsec:project-management}
+The purpose of this project is to investigate and implement a mock rover.
+Because of this the team members must be in the mindset to analyze each part of the project and identify as many problem areas that need answers as possible. 
+This means being specific on how we are going to accomplish our goals.
+We want our team asking the question:"what material, what algorithm, with what method will we be accomplishing this?"
+It is very easy to over estimate the team's ability especially when the project is so new to the team.
+These are skills that benefit RIT SPEX. 
+
+\section{Implementation}
+\label{sec:implementation}
+This project would involve divvying out the different project areas to the different team members or small groups and each week meeting to discuss progress.
+The rover project lends itself well to modularity.
+This will allow us to set up the project in a way that best matches the skillset of the team we have. 
+The team would take this information, refine it and put it into the technical report along with construction. 
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+The deliverable includes the technical report, rover software, and rover hardware.
+The primary deliverables of this project will be the technical report and the rover itself by the end of the second semester.
```

Comment:
Having a formal product requirements document as a deliverable could be very helpful.  It could serve as an early milestone for keeping the design on track and focused.. It may also serve as a good stepping stone when working on the technical report (and should help to explain why certain decisions were made).

What are other people's thoughts on this?